pkg/systemd/services/service-details.jsx

```diff
@@ -208,11 +208,40 @@ export class ServiceDetails extends React.Component {
             waitsFileAction: false,
             note: "",
             error: "",
+            delay: 30000, // 30s
+            unit_properties: {},
         };
 
         this.onOnOffSwitch = this.onOnOffSwitch.bind(this);
         this.unitAction = this.unitAction.bind(this);
         this.unitFileAction = this.unitFileAction.bind(this);
+        this.addUnitProperties = this.addUnitProperties.bind(this);
+        this.doPropertyPolling = this.doPropertyPolling.bind(this);
```

Comment:
Not necessary -- these are always called directly, not as a "function pointer" to component properties.